cmd/harmony/config_migrations.go

```diff
@@ -0,0 +1,81 @@
+package main
+
+import (
+	"errors"
+
+	"github.com/pelletier/go-toml"
+)
+
+func migrateConf(confBytes []byte) (harmonyConfig, string, error) {
+	confTree, err := toml.LoadBytes(confBytes)
+	if err != nil {
+		return harmonyConfig{}, "", err
+	}
+
+	confVersion := confTree.Get("Version").(string)
+	if confVersion == "" {
+		return harmonyConfig{}, "", errors.New("invalid config file - version missing")
+	}
+	migratedVer := confVersion
+	migration, found := migrations[confVersion]
+	// Legacy config versions
+	if !found {
+		migration = migrations["1.0.4"]
```

Comment:
Probably show some warning message in this case?

Reply:
Will add a warning message